data_processors/pipeline/lambdas/update_google_lims.py

```diff
@@ -0,0 +1,197 @@
+from datetime import datetime
```

Comment:
Need insert this decor at top of the module to make it as a Standalone Django app; so that to deploy into Lambda environment.

```
try:
    import unzip_requirements
except ImportError:
    pass

import django
import os

os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'data_portal.settings.base')
django.setup()

# ---

```

Reply:
And also need to add into `serverless.yml` for SLS Lambda deployment descriptor. I reckon, this handler can take similar to one of those lines there:

https://github.com/umccr/data-portal-apis/blob/dev/serverless.yml#L211

Reply:
Fixed in 1cb94eb